### PR TITLE
2462 - IdsDropdown tooltip zindex

### LINKED ISSUE #2462: IdsDropdown: Tooltips don't work when the dropdown is in a modal
**Describe the bug**
Tooltips on the dropdown and dropdown options don't work when the dropdown is in a modal

**To Reproduce**
1. Go to https://stackblitz.com/edit/ids-enterprise-wc-120-b3fpws?file=index.html
2. Hover the dropdown to confirm that the tooltip appears
3. Click on "Trigger a Modal"
4. Hover the dropdown
5. Notice that no tooltip appears

**Expected behavior**
Tooltips should work when the dropdown is in a modal

**Version**
- ids-enterprise: `1.2.0`

**Platform**
 - Infor Application/Team Name: `OS Portal`

### PR DESCRIPTION
**Pull request by jdilag90**
**Explain the details for making this change. What existing problem does the pull request solve?**
Update z-index of tooltip popups.
IdsModal z-index is hard coded to `1020` [here](https://github.com/infor-design/enterprise-wc/blob/c1144f4bbafd06137675ba9bbbeb65cc71594754/src/components/ids-modal/ids-modal.ts#L56).
I updated popup[type='tooltip'] css z-index to `1022`

**Related github/jira issue (required)**:
Closes #2462 

**Steps necessary to review your pull request (required)**:
1. Checkout branch
2. Go to http://localhost:4300/ids-dropdown/in-action-panel.html
3. Open modal, hover over dropdown
4. Check that tooltip appears

**Included in this Pull Request**:
- [x] A note to the change log.

### CODE REVIEW

File: src/components/ids-popup/ids-popup.scss [hunk 1]
```diff
@@ -8,6 +8,10 @@
   text-align: unset;
 }
 
+:host([type='tooltip']) {
+  z-index: 1022;
```
Comment (tmcconechy) [inline src/components/ids-popup/ids-popup.scss:12]: @jdilag90 from my spread sheet of zindex tested maybe this one should be 6000

```
20 - Contextual Toolbar
500 - Side Menu, Status Bar
600 - App Menu, Module Nav
1000 - Overlay
1010 - Busy Indicator
1015 - Popdown
1020 - Dialogs (Modals)
3000 - Popovers 
4000 - Lookup
4500 - Drop Down
6000 - Tooltips
7000 - Draggables
8000 - Slide in Message Dialog/Toast
9000 - Sticky Nav / Context Menus
10000 - Full Page Sticky Nav, Search Results
10020 - Skip link
```

Comment (jdilag90) [reply src/components/ids-popup/ids-popup.scss:12]: @tmcconechy that's a useful list. Should those be tokenized?

Comment (tmcconechy) [reply src/components/ids-popup/ids-popup.scss:12]: @jdilag90 sure good idea

**Review by tmcconechy — APPROVED** [review]

**Comment by tmcconechy** [comment]
@n-ace-ancog i had to merge this for the release. Can you just test it on main and see if any issues?

**Pull request merged**